README.md

```diff
@@ -1,21 +1,52 @@
 # Overview
-Act component is an active component that sends a message to Connect by MQ, waits for a response message for a specified timeout, and returns the received message to the caller. If no response message is received, Act returns a gRPC error response to the script. Communication with Connect  takes place via a message queue, communication with the script - via gRPC. 
 
-Act sends events about the status of the internal steps to the "Event store" component during request processing. These events are written to the root event associated with this request to Act and, as a result, get into the report on the script run.
+Act is a passive th2 component with parameterised functions which is implemented as parts of the logic test. Script or other components can call these functions via gRPC.
```

Comment:
Act is a passive th2 component with **parameterized functions** which is implemented **as part of the test logic**. Script or other components can call these functions via gRPC.